cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -162,6 +164,7 @@ MSG(neighbor_count_lt_one, "Neighbor count lower than one")
 MSG(unknown_distance_type,
     "Custom distances for k-NN is not supported, use one of the predefined distances instead.")
 MSG(distance_is_not_supported_for_gpu, "Only Euclidean distances for k-NN is supported for GPU")
+MSG(invalid_set_of_result_options_to_search, "Invalid set of optional results to search")
```

Comment:
Suggest to figure out more general error message, otherwise you'll need to create new error messages for all kind of tasks

Reply:
Fixed and discussed